db/schema.rb

```diff
@@ -10,15 +10,15 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20210326232803) do
+ActiveRecord::Schema.define(version: 20210326223431) do
 
-  create_table "authors", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=latin1" do |t|
+  create_table "authors", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
```

Comment:
Oh, I didn't realize my local DB had a different charset, should I update it and run the migrations again to re-generate this file and avoid the unnecessary diff? @mobitar

Reply:
UTF8 is the better encoding here anyway so I think it's fine 👍